ert3/config/_stages_config.py

```diff
@@ -1,9 +1,26 @@
 import importlib
 import os
+import sys
 from typing import List, Callable, Optional
 from pydantic import root_validator, validator, FilePath, BaseModel
 
 
+def _import_from(path):
+    if ":" not in path:
+        raise ValueError("Function should be defined as module:function")
+    module, func = path.split(":")
+    try:
+        sys.path.append(os.getcwd())
```

Comment:
It can be removed after loading the module but that is not needed it will only be part of sys paths only during the runtime of the application.... But now I remember the discussion regarding possibly polluting the module namespace. 

I will remove the path after loading the module.

Reply:
I was just thinking that since cloudevent is all crazy, we might end up with elements on the path on the compute node that is not available on disk anymore or similar odd situations 🤷

Reply:
Also, the `os.getcwd()` breaks the current social promise that any `ert3` command should succeed from any place within the workspace. However, if we want to keep this, we should make tests for it. Perhaps you could make an issue and point to this line to at least highlight the discussion?

Reply:
Not really sure what "social promise" you are referring to as far as I can tell `ert3 run` in the current example only works when running from the workspace root, but thanks for pointing it out I fixed the PR so when evaluation a function it will work from anywhere in the workspace

Reply:
You are right! It used to work though, and everything outside loading the configuration should be tooling on top of the `workspace_root`, which is identified by moving up in the directory structure looking for `.ert`. But, the parsing of the configuration files are currently not done relative to the file.

The reason I named it a "social promise" was that it was an idea at some point, but as it is neither documented nor tested I have a hard time calling it a feature :)